sirepo_bluesky/__init__.py

```diff
@@ -1,3 +1,12 @@
+import intake
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
+
+# Look up a driver class by its name in registry.
+catalog_class = intake.registry["bluesky-mongo-normalized-catalog"]
+
+sirepo_bluesky_catalog_instance = catalog_class(
+        metadatastore_db="mongodb://localhost:27017/md",
+        asset_registry_db="mongodb://localhosty:27017/ar",
```

Comment:
I actually like the name - `localhosty` :)
```suggestion
        asset_registry_db="mongodb://localhost:27017/ar",
```